components/scg/src/scg-render.js

```diff
@@ -237,11 +237,12 @@ SCg.Render.prototype = {
                         return;
                     if (self.scene.getObjectByScAddr(d.sc_addr) instanceof SCg.ModelEdge)
                         return;
-                    if (self.sandbox.isRrelKeyScElement)
+                    if (self.sandbox.isSceneWithKey)
                         self.sandbox.updateContent(d.sc_addr, self.scene);
                 })
                 .on("dblclick", d => {
-                    if (SCWeb.core.Main.mode === SCgEditMode.SCgModeViewOnly) return;
+                    if (SCWeb.core.Main.editMode === SCgEditMode.SCgViewOnly) return;
+                    if (SCWeb.core.Main.viewMode === SCgViewMode.DistanceBasedSCgView) return;
```

Comment:
I'd rather not disable dblclick for distance based mode but probably check why we don't show scg on dblclick for full screen scg dbl click and distance based as well